geomstats/learning/frechet_mean.py

```diff
@@ -355,3 +366,87 @@ def fit(self, X, y=None, weights=None):
         self.estimate_ = mean
 
         return self
+
+    def transform(self, X, y=None, base_point=None):
+        """Lift data to a tangent space.
+
+        Compute the logs of all data point and reshapes them to
+        1d vectors if necessary. By default the logs are taken at the mean
+        but any other base point can be passed. Any machine learning
+        algorithm can then be used with the output array.
+
+        Parameters
+        ----------
+        X : array-like, shape=[n_samples, {dim, [n, n]}]
+            Data to transform.
+        y : Ignored (Compliance with scikit-learn interface)
+        base_point : array-like, shape={dim, [n,n]}, optional (mean)
+            Point on the manifold, the returned samples will be tangent
+            vectors at the base point.
+
+        Returns
+        -------
+        X_new : array-like, shape=[n_samples, dim]
+        """
+        if base_point is None:
+            base_point = self.estimate_
+
+            if self.estimate_ is None:
```

Comment:
Needs to be unindented?

Reply:
indent was intended: you don't need to fit if base_point is given

Reply:
But what about base_point=self.estimate_ is given? Unlikely?

Reply:
aaaah stg like `mean.transform(X, base_point=mean.estimate_)`? True...

Reply:
But then mean.estimate_ == None so it works?

Reply:
yes, ok!